templates/data/incubator-buildpack-data.yaml

```diff
@@ -263,6 +263,29 @@ templates:
               pipeline.installer: reconciler
               pipeline.test: "fast-integration"
               pipeline.clusterprovisioning: reconciler
+          reconciler_image_autobump_jobConfig:
+            image: "eu.gcr.io/kyma-project/test-infra/kyma-integration:v20211201-ebdeacd8"
+            command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/reconciler-image-autobump.sh"
+            extra_refs:
+              k8s-test-infra:
+                - org: kubernetes
+                  repo: test-infra
+                  base_ref: master
+                  base_sha: "3fba7110c39a7a0e7fb087dbb4af0674cafd341c"
+                  path_alias: github.com/kubernetes/test-infra
+          github_token_mounts:
```

Comment:
Don't we have a global config for kyma-bot that you can inherit?

Reply:
We have a label preset for github-bot which sets the `envs`. But I didn't find any which attaches the bot token as volume.